### PR TITLE
Fixes #13098: GLPI plugin

### PR DESCRIPTION
**Pull request by VictorQrt**
https://www.rudder-project.org/redmine/issues/13098

### CODE REVIEW

File: glpi/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  /opt/rudder/bin/rudderpkg install-file <plugin>.rpkg
+  ```
+
+- Edit the configuration file in /opt/rudder/etc/glpi.conf. This config file must contain the URL to your
+  GLPI server's fusion inventory plugin.
+
+Usage
+-----
+
+Run /opt/rudder/bin/glpi-plugin send-all to update all the inventories. This command should be cronified everyday around 7AM.
```
Comment (peckpeck) [inline glpi/README.md]: You should put this in the postinstall script and remove it from the doc

File: glpi/bin/glpi-plugin [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+INV_N=$(ls -1 $INVENTORIES_DIR | egrep '.ocs$' | wc -l)
+
+send_inventory()
+{
+  curl -H "Content-Type: Application/x-compress" -k -s --data @$1 $GLPI_FUSION_URL > /dev/null
+}
+
+
+case $1 in
+  send-all*)
```
Comment (peckpeck) [inline glpi/bin/glpi-plugin]: why the * ?

File: glpi/bin/glpi-plugin [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+INVENTORIES_DIR=/var/rudder/inventories/received
+GLPI_FUSION_URL=$(sed -e "/GLPI_FUSION_URL/s/^.*=//" /opt/rudder/etc/glpi.conf)
+INV_N=$(ls -1 $INVENTORIES_DIR | egrep '.ocs$' | wc -l)
+
+send_inventory()
+{
+  curl -H "Content-Type: Application/x-compress" -k -s --data @$1 $GLPI_FUSION_URL > /dev/null
```
Comment (peckpeck) [inline glpi/bin/glpi-plugin]: we prefer the form "${VAR}" for variables

Comment (peckpeck) [reply glpi/bin/glpi-plugin]: everywhere

**Comment by VictorQrt** [comment]
Commit modified

**Pull request merged**